sos/plugins/dnf.py

```diff
@@ -60,4 +66,17 @@ def setup(self):
         if self.get_option("history"):
             self.add_cmd_output("dnf history")
 
+        if self.get_option("history-info"):
+            history = self.call_ext_prog("dnf history")
```

Comment:
As I wrote, perhaps it does - not sure if that's considered ok here. @bmr-cymru what do you say?